connectors/s3-tests/src/test/resources/s3test/api_test.bal

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
We should change the name of the bal file.

Reply:
done